library_generation/test/generate_library_unit_tests.sh

```diff
@@ -33,6 +34,36 @@ __assertFileDoesNotExist() {
   return 1
 }
 
+# Convert OS type to OS architecture, the default OS architecture is
```

Comment:
Can you detect the OS in some easy way?

Reply:
I use `uname -sm` to get os information.